### PR TITLE
Android Issue 175

### PR DESCRIPTION
**Pull request by Himanshu-Vishwas**

### CODE REVIEW

**Comment by baghaii** [comment]
@Himanshu-Vishwas Did the template show up when you created this PR? Can you fill out the template and include a screenshot of what your new feature looks like?

Thank you.

File: coding-projects/android/TaskTracker/app/src/main/java/com/example/tasktracker/ui/TaskSettings/TaskSettingsScreen.kt [hunk 1]
```diff
@@ -58,7 +71,31 @@ fun TaskSettingsScreen() {
...
+                        .padding(16.dp),
+                    verticalAlignment = Alignment.CenterVertically
+                ) {
+
+                    // Spacer for visual separation
+                    Spacer(modifier = Modifier.width(16.dp))
+
+                    // Text for Show Badge setting
+                    Text(
+                        text = "SHOW BADGE ON APP ICON",
```
Comment (baghaii) [inline coding-projects/android/TaskTracker/app/src/main/java/com/example/tasktracker/ui/TaskSettings/TaskSettingsScreen.kt:86]: Move this string to the strings.xml file, and pull it in using something like `stringResource(id = R.string.ok)`

In an earlier PR, @gaurigadkari reminded someone not to put the strings in all caps in the string resource file but to use `uppercase()` to convert them later. That seemed like good advice.

File: coding-projects/android/TaskTracker/app/src/main/java/com/example/tasktracker/ui/TaskSettings/TaskSettingsScreen.kt [hunk 2]
```diff
@@ -58,7 +71,31 @@ fun TaskSettingsScreen() {
...
-                // TODO - #175 - Add a "Show Badge" row
+                Row(
+                    modifier = Modifier
+                        .fillMaxWidth()
+                        .padding(16.dp),
+                    verticalAlignment = Alignment.CenterVertically
+                ) {
+
+                    // Spacer for visual separation
+                    Spacer(modifier = Modifier.width(16.dp))
```
Comment (baghaii) [inline coding-projects/android/TaskTracker/app/src/main/java/com/example/tasktracker/ui/TaskSettings/TaskSettingsScreen.kt:82]: I am assuming you are using this spacer to get some space from the left side of the screen. If so, can you do that with the padding on your row and remove the spacer?

padding can take named arguments like start, top, bottom, and end. You can just set start to 32.dp if the goal is to have more space on the left side.

**Comment by renweview** [comment]
@Himanshu-Vishwas How's it going with this? Please ping when the edits are fixed otherwise the pr would close in 48 hours. To follow the contribution protocol, please check out the Slack group so we can work with you on there

**Comment by renweview** [comment]
Hey @Himanshu-Vishwas anyone can contribute but we're encouraging the protocol so everyone has a fair shot at the issues. I know we're over time for this one, do you want a day to update this or should we close?

**Pull request closed without merging**